src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -60,6 +62,34 @@
 logger = get_task_logger(__name__)
 
 
+def get_access_manual_webhook_inputs(
+    db: Session, privacy_request: PrivacyRequest, policy: Policy
+) -> Tuple[Dict[str, List[Dict[str, Optional[Any]]]], bool]:
```

Comment:
Can we get this added as a pydantic type? Specifically, I'm unsure what the `bool` refers to.

Reply:
sure, it's in the docstring, but I see that it's less clear.  the bool is whether or not request execution should proceed, but I can convert this into a pydantic schema.

Reply:
If it's not too much work, that'd be great to have this as a pydantic schema